### PR TITLE
Add loading icon

### PR DESCRIPTION
**Pull request by bhaamati**
I managed to mistype icon. Sorry! This just adds a spinner to the resources pages as the maps loads.

### CODE REVIEW

**Comment by ekwkk** [comment]
Wow, that's really cool!!

File: src/main/java/com/google/codeu/servlets/ClinicDataServlet.java [hunk 1]
```diff
@@ -2,78 +2,55 @@
 
 import com.google.gson.Gson;
 import com.google.gson.JsonArray;
+import com.opencsv.bean.CsvToBean;
+import com.opencsv.bean.CsvToBeanBuilder;
 
 import javax.servlet.annotation.WebServlet;
+import com.google.codeu.data.Clinic;
```
Comment (ekwkk) [inline src/main/java/com/google/codeu/servlets/ClinicDataServlet.java]: Alphabetize `import`s :)

File: src/main/java/com/google/codeu/data/Clinic.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package com.google.codeu.data;
+import com.opencsv.bean.CsvBindByName;
+import java.util.Arrays;
+
+public class Clinic {
```
Comment (ekwkk) [inline src/main/java/com/google/codeu/data/Clinic.java:10]: I think it'll be better to add some comments

File: src/main/webapp/css/main.css [hunk 3]
```diff
@@ -75,52 +82,79 @@ nav li {
 .logo {
   font-size: 170%;
   font-weight: 800;
-  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-family: "Merienda", Verdana, Geneva, Tahoma, sans-serif;
 }
 
 html {
-  background-color: rgba(226, 96, 120, 0.781);
+  /*background-color: rgba(226, 96, 120, 0.781);*/
```
Comment (ekwkk) [inline src/main/webapp/css/main.css]: Delete commented lines

**Review by ekwkk — APPROVED** [review]

File: src/main/java/com/google/codeu/data/Clinic.java (hunk 2 above)
Comment (bhaamati) [reply src/main/java/com/google/codeu/data/Clinic.java:10]: Good suggestion!

**Pull request merged**